tests/test_pyqvm.py

```diff
@@ -154,6 +154,36 @@ def test_hermitian_expectation(self, shots):
 
         self.assertAllAlmostEqual(res, expected, delta=3/np.sqrt(shots))
 
+    def test_multi_mode_hermitian_expectation(self, shots, qvm, compiler):
+        """Test that arbitrary multi-mode Hermitian expectation values are correct"""
+        theta = 0.432
+        phi = 0.123
```

Comment:
Instead of picking hard values like this, how about picking from some range at random and then fixing the seed? This way, it's obvious from the start that `theta` and `phi` are parameters of a more generally applicable expression.

Reply:
Done :+1:

Reply:
Why not do this for all the `theta = 0.432`, `phi = 0.123` definitions in the tests in `test_pyqvm.py`?

Reply:
There are also similar (but different) parameters in `test_one_qubit_wavefunction_circuit` that we could probably similarly replace with randomly picked values.

Reply:
Ah, I didn't spot the others in the diff. Does it make sense to do this in a new PR?

Reply:
Sure, that sounds good to me.